src/osd/OSD.cc

```diff
@@ -3667,6 +3667,18 @@ float OSD::get_osd_recovery_sleep()
     return cct->_conf->osd_recovery_sleep_hdd;
 }
 
+float OSD::get_osd_recovery_sleep_degraded() {
+  float osd_recovery_sleep_degraded = cct->_conf.get_val<double>("osd_recovery_sleep_degraded");
+  if (osd_recovery_sleep_degraded > 0)
```

Comment:
Add braces around the if..else blocks. The other functions returning the configs don't use it but I think for any new code it's better to follow the guidelines.